Create 2020-07-21-why-paper-and-pencil-is-my-best-tool.markdown (#2)
guilhermesiani/guilhermesiani.github.io: _posts/2020-07-21-why-paper-and-pencil-is-my-best-tool.markdown

---
layout: post
title:  "Why paper and pencil are my best tools"
date:   2020-07-21 23:30:41 -0300
categories: tools
---

> Try to think about how much time we spend looking for an ideal tool to taking our notes, reminders, scheduling, sketching… A tool that give us freedom and flexibility to do everthing we need. Yeah .. so try to look around. Can you see this tool? There it is! A simple paper and pencil right next to you.

Reviewer: ```suggestion
Just try thinking about how much time we spend looking for an ideal tool to taking our notes, reminders, scheduling, sketching… A tool that give us freedom and flexibility to do everything we need. Yeah... so try looking around. Can you see this tool? There it is! A simple paper and pencil right next to you.
```

> I can’t imagine how much ideas and time I saved using this and I will try explain why this works so well to me.

Reviewer: ```suggestion
```

Author: ```suggestion
I can’t imagine how many ideas and how much time I saved using this and I will try explain why this works so well to me.
```

> All my programmer life I’ve tried to use many tools that improve my capabilities to remember things, organize TODO lists, improve creativity and etc. One thing that works so well on all my needs, with freedom and combined all other tools that have separated and specialized functions, is the paper and pencil.

Reviewer: ```suggestion
In all my programmer life I’ve tried to use many tools that could improve my capabilities to remember things, organize to do lists, improve creativity, etc. One thing that worked so well on all my needs giving me freedom, and combining all other tools that have separated and specialized functions, is paper and pencil.

I can’t imagine how many ideas and how much time I've saved using just this tool, and I will try explain why this works so well to me.
```

> > Ok, Guilherme. But what about smartphone screen notifications for your schedule todo list/meeting?

Reviewer: ```suggestion
> Ok, But what about smartphone's on-screen notifications for your scheduled to do lists/meetings?
```

> Ok. I know and I use this too as a extra (Yeah.. as a extra, not as the main tool)

Reviewer: ```suggestion
Ok. I know and I use this too as an extra (Yeah... as an extra, not as the main tool)
```

> Organizing into paper or a notebook turns me more disciplined ever, just because I can create mindsets, draw ever I need to explore more and when I must remember some topics with emfhasis. This kind of things enforces our memory so well that tecnologies will never do the same.

Reviewer 2: Typos: "Draw ever", "enfhasis".

Author: I didn't understand. Google translate tells me that is ok with these words... Is it really wrong?

> Organizing into paper or a notebook turns me more disciplined ever, just because I can create mindsets, draw ever I need to explore more and when I must remember some topics with emfhasis. This kind of things enforces our memory so well that tecnologies will never do the same.

Reviewer: ```suggestion
Organizing ideas on a paper or a notebook turned me more disciplined than ever, just because I can create mindsets, draw whatever I need to explore and remember myself about some topics with emphasis. This kind of things enforces our memory so well that there will never be a technology to do it in the same way
```

![image](/assets/images/creativity-paper-1280.jpg)

> Although tecnologies has credits about much other things and we are programmers because of this, I could’n abandon paper and freedom.

Reviewer: ```suggestion
Although technology has its credits about many other things, and we are programmers because of it, I couldn't abandon paper and its freedom.
```

> I did many class diagrams, flowcharts and database modeling that saved me much time rather start coding too fast, without thinking better what I really could do.

Reviewer: ```suggestion
I made many class diagrams, flowcharts and database modeling which saved me much time, instead of start coding right away, without thinking carefully of what I could really do.
```

> This experience demonstrate that this action save me much more time than if I had coded first and thinking how to solve my problem or applying some design patterns simultaneos. It is right that easier things we just code directly. Trying to draft this on paper will cost more time that codefying and that’s fine.

Reviewer 2: Suggestions:
"simultaneous" -> "simultaneously"
"easier things" -> "easy things"
"codifying" -> "coding"

> This experience demonstrate that this action save me much more time than if I had coded first and thinking how to solve my problem or applying some design patterns simultaneos. It is right that easier things we just code directly. Trying to draft this on paper will cost more time that codefying and that’s fine.

Reviewer: ```suggestion
This past experience demonstrates that this action saves me much more time than if I had coded first and just then thought on how to solve my problem or applying any design patterns. It's OK to code directly if it's something easy, trying to draft this on paper could cost more time than code directly and that’s fine.
```

> Try it and tell me later about your experience.

Reviewer: ```suggestion
Try it, and tell me later about your experiences.
```

### Bullet Journal

> Another big change that I did on my life organization methods is the use of Bullet Journal and I recomMend you to know about it a bit.

Reviewer 2: Typo: "recomMend"

> Another big change that I did on my life organization methods is the use of Bullet Journal and I recomMend you to know about it a bit.

Reviewer: ```suggestion
Another big change I've made on my life's organization methods is the use of Bullet Journal and I recommend you to know a bit about it.
```

![image](/assets/images/bulletjournal.png)

> Bullet Journal helps me bringing my thoughts closer. For a long time I don't feel what is do something more ~~horrible~~ artistic and handmade. Even if simplistic, all that I said above abou paper and pencil is more powerful using Bullet Journal (In this case I use a pen :p).

Reviewer 2: Typo: "abou"

> Bullet Journal helps me bringing my thoughts closer. For a long time I don't feel what is do something more ~~horrible~~ artistic and handmade. Even if simplistic, all that I said above abou paper and pencil is more powerful using Bullet Journal (In this case I use a pen :p).

Reviewer: ```suggestion
Bullet Journal helps me bringing my thoughts closer. For a long time I didn't feel what it is to do something ~~horrible~~ more artistic and handmade. Even if simplistic, all that I said above about paper and pencil is more powerful along with Bullet Journal (for this I use a pen :p).
```

> I put on this notebook my year journey. Every month, every single day, all my notes, todo lists and reminders in one place is a kind of freedom that no app gave me.

Reviewer: ```suggestion
I put down on this notebook my year journey. Every month, every single day, all my notes, to do lists and reminders, all in one place, this is a kind of freedom that any app gave me.
```

> Don’t stick to the stereotype that programmers should consume all possible technology.

Reviewer: ```suggestion
Don’t stick with the stereotype that programmers should consume every possible technology that there is.
```

> I hope this experience I shared at least make you reflect.

Reviewer: ```suggestion
I hope the experience I shared, at least makes you think about it.
```

---
layout: post
title:  "Why paper and pencil are my best tools"
date:   2020-07-21 23:30:41 -0300
categories: tools
---

> Just try thinking about how much time we spend looking for an ideal tool to taking our notes, reminders, scheduling, sketching… A tool that give us freedom and flexibility to do everything we need. Yeah... so try looking around. Can you see this tool? There it is! A simple paper and pencil right next to you.

Reviewer 3: May you should use "Just try to think"

I can’t imagine how many ideas and how much time I saved using this and I will try explain why this works so well to me.

> In all my programmer life I’ve tried to use many tools that could improve my capabilities to remember things, organize to do lists, improve creativity, etc. One thing that worked so well on all my needs giving me freedom, and combining all other tools that have separated and specialized functions, is paper and pencil.

Reviewer 3: The socond frase sounds a little weird, there is nothing wrong, but doesn't look naturally in english: 
"One thing that worked so well on all my needs giving me freedom..."
What do you think about?
"The one that's meet all my needs and give me freedom..."

Author: Nice one. I apply some fixes in your suggestion too:

`The one that meets all my needs and gives me the freedom`

I can’t imagine how many ideas and how much time I've saved using just this tool, and I will try explain why this works so well to me.

> Ok, But what about smartphone's on-screen notifications for your scheduled to do lists/meetings?

Ok. I know and I use this too as an extra (Yeah... as an extra, not as the main tool)

> Organizing ideas on a paper or a notebook turned me more disciplined than ever, just because I can create mindsets, draw whatever I need to explore and remember myself about some topics with emphasis. This kind of things enforces our memory so well that there will never be a technology to do it in the same way

Reviewer 3: What you think about instead of:
"remember myself about some topics with emphasis."
use:
"remember myself emphasizing some topics."
?

![image](/assets/images/creativity-paper-1280.jpg)

Although technology has its credits about many other things, and we are programmers because of it, I couldn't abandon paper and its freedom.

I made many class diagrams, flowcharts and database modeling which saved me much time, instead of start coding right away, without thinking carefully of what I could really do.

This past experience demonstrates that this action saves me much more time than if I had coded first and just then thought on how to solve my problem or applying any design patterns. It's OK to code directly if it's something easy, trying to draft this on paper could cost more time than code directly and that’s fine.

Try it, and tell me later about your experiences.

### Bullet Journal

> Another big change I've made on my life's organization methods is the use of Bullet Journal and I recommend you to know a bit about it.

Reviewer 3: "I recommend you to know a bit about it."
should be:
"I recommend knowing a bit about it."
